napalm_eos/eos.py

```diff
@@ -204,18 +204,26 @@ def rollback(self):
         commands.append('write memory')
         self.device.run_commands(commands)
 
+    def _get_version(self):
```

Comment:
> result = self.device.run_commands(commands)[0]

No particular reason, just my personal preference - I don't like when there are many parans :)

> And it also forces you to do several API calls instead of making one with several commands (particularly painful if you are operating devices with a high latency).

In ```get_facts``` right? That's true, will change that.